src/pages/planning-applications/[id]/index.tsx

```diff
@@ -34,7 +34,7 @@ export async function getStaticPaths() {
   const data = await getActiveApplications();
 
   return {
-    paths: data.map((doc: any) => ({ params: { data: doc, id: doc._id } })),
+    paths: data?.map((doc: any) => ({ params: { data: doc, id: doc._id } })),
```

Comment:
what does paths contain in this scenario (when data is empty), does it just produce an empty path array?
Do we need to explicitly handle this?
`if (!data || data.length === 0) {
    return {
      paths: [],
      fallback: "blocking",
    };
  }`